backend/Events/LobbyEvents.js

```diff
@@ -0,0 +1,27 @@
+const Player = require("../domain/Player");
+const LobbyCodeDTO = require("../domain/DTO/response/LobbyCodeDTO");
+const MafiaGame = require("../domain/MafiaGame");
+
+/**
+ * Event handlers and logic for `create-lobby` and `lobby-code`
+ * The goal of these lobby events is to allow a host to create a game and receive a new room id. 
+ * @param {any} io 
+ * @param {any} socket 
+ * @param {MafiaGame} mafiaGame 
+ */
+exports.load_lobby_events = function(io, socket, mafiaGame){
```

Comment:
why is this exports.`load_lobby_events`

Reply:
I think you can use `exports.load_lobby_events` :) In this case, it should work, I did some googling here: https://blog.tableflip.io/the-difference-between-module-exports-and-exports/. However, we could just stick to `module.exports` so we can avoid potential errors as highlighted in that article / be consistent.

Reply:
Yeah, I _think_ you can do module.exports.functionName = blahblah for other event handlers you want to add in this file :)